src/main/resources/db/migration/V1_8__ADD_SOFT_DELETE.sql

```diff
@@ -0,0 +1,4 @@
+alter table region add column soft_deleted boolean default false;
```

Comment:
It's a big change and probably not worth changing, but when I have seen this done elsewhere they have used the `active` flag. I don't think it's a convention either so whatever....

Reply:
yeah, depends on the context, I've seen active and inactive mean something different to soft deleted, going in and out of those states meant different processes